app/presenters/two_factor_authentication/set_up_selection_presenter.rb

```diff
@@ -2,7 +2,7 @@ module TwoFactorAuthentication
   class SetUpSelectionPresenter
     include ActionView::Helpers::TranslationHelper
 
-    attr_reader :user
+    attr_reader :user, :configuration
```

Comment:
I wouldn't expect there should ever be a `configuration` for a SetUp presenter, since my understanding is that the key distinction between the two types is whether the MFA is already enrolled (i.e. a configuration exists).